src/lib/cimb_translator/CimbReader.cpp

```diff
@@ -92,6 +92,17 @@ namespace {
 		decoder.update_color_correction(color_correction::get_adaptation_matrix<adaptation_transform::von_kries>(white, {255.0, 255.0, 255.0}));
 		return true;
 	}
+
+	// we will always skip the last unaltered block of a file
+	// if it does not exactly match the chunk size
+	// -- specifically, the encoder will skip *any* block that does not
+	// match, but it so happens that it's only ever that one block.
+	unsigned computeRadioactiveBlockId(const FountainMetadata& md, unsigned chunk_size)
+	{
+		if (md.file_size() % chunk_size == 0)
+			return 0xFFFFFFFF; // there isn't one
```

Comment:
If there isn't a missized-block, this serves as a dummy value we'll never hit. (block_id is a 16 bit field)